src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -499,16 +491,12 @@ private <T> List<ReferenceSetMember> updateMetadataRefset(Map<String, Set<String
 							.noneMatch(id -> id.equals(member.getMemberId()))) {
 
 				member.setActive(false);
-				if (!member.buildReleaseHash().equals(member.getReleaseHash())) {
-					member.markChanged();
-				}
-				
-				//Any change to a component in an extension needs to be done in the default module
+				member.markChanged();
```

Comment:
Are we definitely sure that only changed members will appear through this code path?   What about if we've changed it back so that it now equals the previous release hash?

Reply:
I think the logic about whether a component has changed is fine even though this method requires refactoring as well. However what is the correct module id to use below from line 505 won't work when reverting extension changes for core module components.